lib/cosmos/gui/qt_tool.rb

```diff
@@ -45,10 +45,37 @@ def initialize(options)
       @options = options
       @about_string = nil
 
+      base_class = self.class.to_s.split('::')[-1].downcase
+      config_dir = File.join(Cosmos::USERPATH, 'config', 'tools', base_class)
+      if File.exist? config_dir
+        @options.config_dir = config_dir
+        options.config_file = verify_config(@options.config_file, ".txt", base_class)
+        options.stylesheet = verify_config(@options.stylesheet, ".css", base_class)
+      end
+
       self.window_title = options.title
       Cosmos.load_cosmos_icon
     end
 
+    def verify_config(file, type, base_class)
+      return file if file && File.exist?(file)
+      if file
+        # Add the configuration dir onto the filename. If this results in a bad
+        # configuration file the config file parsing code will notify the user.
+        File.join(@options.config_dir, file)
+      else
+        # No file passed so default to a file named after the class
+        config_file = File.join(@options.config_dir, "#{base_class}#{type}")
+        unless File.exist?(config_file)
+          # Grab the first file sorted alphabetically
+          # This could result in nil which is ok meaning no configuration file
+          config_file = Dir[File.join(@options.config_dir, "*#{type}")].sort[0]
+        end
```

Comment:
Ok but I still will go look for a css named after the tool, e.g. launcher.css which will apply by default even if they don't specify it on the command line. I think that's a meaningful default and makes it easy to apply styles without having to modify your launcher config to specify each css file.